src/file.rs

```diff
@@ -205,71 +238,92 @@ impl AsepriteFile {
             if !self.layer(layer_id).is_visible() {
                 continue;
             }
-            self.copy_cel(&mut image, cel);
+            self.write_cel(&mut image, cel);
         }
 
         image
     }
 
-    fn copy_cel(&self, image: &mut RgbaImage, cel: &RawCel) {
+    fn write_cel(&self, image: &mut RgbaImage, cel: &RawCel) {
         assert!(self.pixel_format != PixelFormat::Grayscale);
```

Comment:
Ok, I see this function contains a bunch of asserts and panics. The reasoning was that all those invariants should be caught by `validate` so they are actually impossible for users. Still we have to double check that that's actually the case.